ansible/inventories/whisk-testnet-0/group_vars/all/all.yaml

```diff
@@ -17,7 +17,7 @@ ethereum_network_deposit_contract_block: >-
 
 ethereum_node_rcp_hostname: "rpc.{{ server_fqdn }}"
 ethereum_node_beacon_hostname: "bn.{{ server_fqdn }}"
-dora_the_explorer_hostname: "beaconlite.{{ ethereum_network_name }}.ethpandaops.io"
+dora_hostname: "beaconlite.{{ ethereum_network_name }}.ethpandaops.io"
```

Comment:
might wanna change this to dora too ?

Reply:
Hmm, I'm not sure about this.
The link has been shared on twitter in context of a whisk testnet announcement.
Changing it would invalidate the old link :/

Reply:
thanks for noticing this! indeed would like to keep the link alive.